src/NuGet.Clients/PackageManagement.UI/Resources.resx

```diff
@@ -177,6 +177,9 @@
   <data name="Label_Options" xml:space="preserve">
     <value>Options</value>
   </data>
+  <data name="Label_Owners" xml:space="preserve">
+    <value>Owner(s): </value>
```

Comment:
I copied and modified Label_Authors, which appears to be used in two places. (also the licensing dialog)
I'll remove the trailing space in Label_Owners.